frontend/src/components/MailingForm.vue

```diff
@@ -12,25 +12,26 @@
     <v-col class="form-box">
```

Comment:
To test the contents of the components once only and for other pages just make sure it is there put a data-cy tag with this.